playbook.md

```diff
@@ -971,87 +971,70 @@ to build an appropriate hosting platform for their service. This is fair
-we are working on a development sprint or on a relevant ticket. We think this
+we are working during scheduled development time or on a relevant ticket. We think this
 is really important.
 
 #### Support rota
 
-We work on tickets by having a developer on support according to a rota.
-Developers are assigned to support for one week at a time and work on tickets
-for the whole week. If there is a ticket they need another developer to help
-with, this work is scheduled.
+We work on tickets by having a developer on support according to a rota. Developers are assigned to support for one week at a time and work on tickets for the whole week. If there is a ticket they need another developer to help with, this work is scheduled.
 
 #### Client experience (AKA: ticket principles)
 
 ##### Be responsive
 
-Clients expect us to deal with their issues promptly. But they understand that
-this isn't always possible. They are generally forgiving of the fact that we're
-sometimes busy, and they understand that some issues are complex and require
-long investigations.
+Clients expect us to deal with their issues promptly. But they understand that this isn't always possible. They are generally forgiving of the fact that we're sometimes busy, and they understand that some issues are complex and require long investigations.
 
-The thing most clients value above all else is being kept informed of what is
-going on. The first quality of a good ticket experience is responsiveness: we
-keep clients informed of what we're doing, even if there hasn't been much
-progress.
+The thing most clients value above all else is being kept informed of what is going on. The first quality of a good ticket experience is responsiveness: we keep clients informed of what we're doing, even if there hasn't been much progress.
 
 ##### Stick to your commitments
 
-It's really important that we do what we say we'll do, and don't promise things
-we can't deliver. If we're unable to deal with a ticket in good time and leave
-an update saying we'll work on it tomorrow, we must meet that commitment.
+It's really important that we do what we say we'll do, and don't promise things we can't deliver. If we're unable to deal with a ticket in good time and leave an update saying we'll work on it tomorrow, we must meet that commitment.
 
-It is doubly bad to fail to meet a commitment and not say anything about it.
-Responsiveness is always the priority. So if for some reason we couldn't do what
-we said we'd do, we always respond to say so.
+It is doubly bad to fail to meet a commitment and not say anything about it. Responsiveness is always the priority. So if for some reason we couldn't do what we said we'd do, we always respond to say so.
 
 ##### Make a good impression
 
-In tickets as in all things, we are mindful of dxw's [values](#values).
+In tickets as in all things, we are mindful of dxw's [values](http://playbook.dxw.com/#/?id=values).
```

Comment:
Any reason why this is the only anchor you've expanded to an absolute link? Was it broken in a way that the other internal anchors weren't?